OJP/OJP_JourneySupport.xsd

```diff
@@ -243,9 +251,9 @@
 			<xs:element name="Extension" type="xs:anyType" minOccurs="0"/>
 		</xs:sequence>
 	</xs:complexType>
-	<xs:complexType name="TripPlaceStructure">
+	<xs:complexType name="TripLocationStructure">
 		<xs:annotation>
-			<xs:documentation>A trip place represents the current logical position of a  journey service. It can be used similarly to a place e.g. for starting a new trip requests from within this service. A trip place does not(!) describe the relative position of a traveller within a vehicle, e.g. the seat.</xs:documentation>
+			<xs:documentation>A trip location represents the current logical position of a  journey service. It can be used similarly to a place e.g. for starting a new trip requests from within this service. A trip location does not(!) describe the relative position of a traveller within a vehicle, e.g. the seat.</xs:documentation>
```

Comment:
```suggestion
			<xs:documentation>A trip location represents the current logical position of a  journey service. It can be used similarly to a place e.g. for starting a new trip request from within this service. A trip location does not(!) describe the relative position of a traveller within a vehicle, e.g. the seat.</xs:documentation>
```

Reply:
Perhaps I miss something - but I don't see how the three elements in the sequence could indicate "the current logical position of a journey service".  I would rather expect elements for current or next call, or vehicle position to do that.